src/room/InCallView.tsx

```diff
@@ -504,7 +502,14 @@ function useParticipantTiles(
             : false;
 
         const id = sfuParticipant.identity;
-        const member = matrixParticipants.get(id);
+        const member = findMatrixMember(matrixRoom, id);
+        // We always start with a local participant wit the empty string as their ID before we're
+        // connected, this is fine and we'll be in "all ghosts" mode.
+        if (id !== "" && member === undefined) {
+          logger.warn(
+            `Ruh, roh! No matrix member found for SFU participant '${id}': creating g-g-g-ghost!`
+          );
+        }
```

Comment:
Is this the behaviour we want? I feel like we could also treat any sfu participant who is not part of the room state as nonexistent. Basically treating it like this: RoomState > sfu participants. If someone wants to be part of a rtc session the most important step is to get oneself into the room state, so that the matrix authentication/identity system is always the source of truth.

Reply:
Potentially, but it's not what we've been doing up until now: this just continues doing what we were doing but logging it explicitly.